internals-js/src/specs/sourceSpec.ts

```diff
@@ -0,0 +1,146 @@
+import { DirectiveLocation, GraphQLError } from 'graphql';
+import { FeatureDefinition, FeatureDefinitions, FeatureUrl, FeatureVersion } from "./coreSpec";
+import {
+  Schema,
+  NonNullType,
+  InputObjectType,
+  InputFieldDefinition,
+  ListType,
+} from "../definitions";
```

Comment:
Nit, but it would be nice to be consistent about single/double quotes within a file.

Reply:
https://github.com/apollographql/federation/pull/2894/commits/928b2942028fcd0709d6a537b4443c061690da27